packages/renovate-config/README.md

```diff
@@ -0,0 +1,110 @@
+# `@lmc-eu/renovate-config`
+
+> LMC's config for renovate
+
+## Usage
+
+Enable Renovate in your repo and just `extends` in `renovate.json`.
+
+```json
+{
+  "extends": ["@lmc-eu"]
+}
+```
+
+Note: You don't have to do `npm i -D @lmc-eu/renovate-config`.
```

Comment:
Magic! 🪄 👀

Reply:
Yes, it searches for `@lmc-eu/renovate-config` on npm. Or it can search for `renovate-config` repo on GitHub.